### DISCUSSION TITLE
Fehler 47?

### DISCUSSION THREAD

**Discussion by 3aau** [hoylabs/OpenDTU-OnBattery#254] [General]
moin, ich habe hier einen fehler 47, ist der schon erklärbar?

**Comment by madmartin** [comment]
Bitte die Frage etwas präziser fomulieren. Wo tritt das auf, vielleicht eine Hardcopy dazu?

**Reply by spcqike** [reply]
wahrscheinlich im Log des Inverters.
https://github.com/tbnobody/OpenDTU/discussions/984

vermutlich hat Fehler 47 etwas mit dem AC Netz zu tun. Ob nun zu hohe Spannung, schlechte Frequenz, kurzzeitiger Ausfall, ... weiß man scheinbar nicht offiziell. wurde von Hoymiles nicht dokumentiert oder herausgegeben.
